portindex2json/portindex2json.tcl

```diff
@@ -12,6 +12,74 @@
 
 package require json::write
 
+proc format_strings {strings} {
+    set new_strings {}
+    foreach element $strings {
+        lappend new_strings [::json::write string $element]
+    }
+    return $new_strings
+}
+
+proc get_maintainer_array {maintainer} {
+    set detail_list [split $maintainer @]
+    if {[string first : [lindex $detail_list 0]] != -1} {
+        set email_list [split [lindex $detail_list 0] :]
+        set name [string trimright [lindex $email_list 1]]
+        set email(name) "\"$name\""
+        set email(domain) "\"[lindex $email_list 0]\""
+        ::json::write indented false
```

Comment:
These intermediate identations mess up the global intentation of the file and make the spacing inconsistent in general. I'm kind of inclined to remove them altogether.

What we could (and probably should) do is provide a global option to switch between the readable and the compact form of the output. Any software would probably want the compact form for everything, not just emails. And any human being who wants to read the long form would not mind having the person split into 6 lines.

Reply:
I have removed the part that was writing maintainers object into one line.

For the global part we just need to add `::json::write intended false`.